lambdatimes/src/components/Header.js

```diff
@@ -1,15 +1,47 @@
 import React from 'react';
+import styled from 'styled-components';
 
 // Refactor this component to use styled components and not classNames. 
 // You can find the corresponding CSS in the CSS/index.css file
 
+const HeaderStyled = styled.div`
+  div {
```

Comment:
Great job nesting these in the same file making this component pretty re-usable if you wanted to use something similar in another project.